packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,14 +137,25 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values likes strings, numbers, and booleans
+* Primitive values
+* Non-rendering values
 * `TemplateResult` objects created with the `html` function.
 * DOM nodes
 * Arrays or iterables of any of the supported types
 
 ### Primitive values
 
-Primitives values like strings, numbers, booleans, null, and undefined are converted to strings when interpolated into text content or attribute values. They are checked for equality with the previous value so the DOM is not updated if the value hasn't changed.
+Strings, numbers, and booleans are all converted to strings when interpolated into text content.
+
+A value of `5` will render the string `'5'`. The empty string `''` is specially treated as a [non-rendering value](#non-rendering).
+
+A boolean value `true` will render `'true'`, and `false` will render `'false'`. Rendering a boolean like this is uncommon. Instead booleans are typically used in conditionals to render other appropriate values. For more on conditionals, see [Conditionals](/docs/templates/conditionals/).
+
+### Non-rendering values {#non-rendering}
```

Comment:
"Non-rendering" sounds to my ear like these have no action, or it describes a set of values that could include `noChange`. These values do render, they just render "nothing".

I would consider a phrase like "Empty values" or "Clearing values". React uses the word "skip" to describe how it handles `false`, but I don't think that quite right.

Reply:
I think that non-rendering is clearer than "empty values" or "clearing values."  "Values that render nothing" is maybe? clearer but also longer. Next choice maybe "Empty values"?

I think for clarity it might be good to mention `noChange` . Maybe something like:

If you write custom directives, you'll encounter another sentinel value, `noChange`. The `noChange` sentinel is used to leave the previously-rendered value in place. For more information, see [Signaling no change](/docs/templates/custom-directives/#signaling-no-change).

Reply:
Attempted to address feedback, ptal.